plugins/rhacm/src/components/ClusterStatusPage/ClusterStatusPage.tsx

```diff
@@ -85,6 +86,11 @@ const CatalogClusters = () => {
       });
       const clusters = await getClusters(configApi);
 
+      if ('error' in clusters) {
+        // Didn't find a better way to do this
```

Comment:
The better way is to check status code instead of parsing the response :wink: 

```ts
try {
  const clusters = await getClusters(configApi)
} catch {
...
}
```

Should do the trick I think.

Reply:
I was referring to the type conversions, but I don't think the try/cache will help since if an error happens in the API call from ACM it returns an error body as the contents and doesn't throw an actuall `Error`. If some other error happens where it is actually thrown, the error handler at line 110 will take care of it.

That is why the error body from the ACM API call is thrown if the resulting cluster variable contains an error key. 

Do you think this is the correct approach or should this be changed so that the API call actually throws an error?